rootdir/root/init.qcom.rc

```diff
@@ -460,7 +424,6 @@ service qmuxd /system/bin/qmuxd
 
 service netmgrd /system/bin/netmgrd
     class main
-    disabled
```

Comment:
I agree, the same goes for ril-daemon and qmux services, they should also be started from main. It looks like init.class_main.sh isn't processed at all